### ISSUE TITLE
NSBluetoothPeripheralUsageDescription required by apple

### ISSUE THREAD

**Issue by peterpaulis** [malcommac/SwiftLocation#334]
Apple requires NSBluetoothAlwaysUsageDescription when using SwiftLocation :(

```
Dear Developer,

We identified one or more issues with a recent delivery for your app, "Footourist - Discover places" 1.0.0 (1). Please correct the following issues, then upload again.

ITMS-90683: Missing Purpose String in Info.plist - Your app's code references one or more APIs that access sensitive user data. The app's Info.plist file should contain a NSBluetoothPeripheralUsageDescription key with a user-facing purpose string explaining clearly and completely why your app needs the data. Starting Spring 2019, all apps submitted to the App Store that access user data are required to include a purpose string. If you're using external libraries or SDKs, they may reference APIs that require a purpose string. While your app might not use these APIs, a purpose string is still required. You can contact the developer of the library or SDK and request they release a version of their code that doesn't contain the APIs. Learn more (https://developer.apple.com/documentation/uikit/core_app/protecting_the_user_s_privacy).

Though you are not required to fix the following issues, we wanted to make you aware of them:

ITMS-90683: Missing Purpose String in Info.plist - Your app's code references one or more APIs that access sensitive user data. The app's Info.plist file should contain a NSBluetoothAlwaysUsageDescription key with a user-facing purpose string explaining clearly and completely why your app needs the data. Starting Spring 2019, all apps submitted to the App Store that access user data are required to include a purpose string. If you're using external libraries or SDKs, they may reference APIs that require a purpose string. While your app might not use these APIs, a purpose string is still required. You can contact the developer of the library or SDK and request they release a version of their code that doesn't contain the APIs. Learn more (https://developer.apple.com/documentation/uikit/core_app/protecting_the_user_s_privacy).

Best regards,

The App Store Team
```

**Comment by arthurdapaz** [comment]
Yep. just forked and removed this feature. Bluetooth is being used for beacon propagation and scanning

**Comment by malcommac** [comment]
Are you talking of the demo project? Library itself does not have it.

**Comment by hamzaozturk** [comment]
Since the library using CoreBluetooth framework for Beacons ranging, monitoring, Apple requires this Purpose String.

I think, we should separate the Beacon feature from the Core and serve it as a sub package under SwiftLocation.

**Comment by itruf** [comment]
@hamzaozturk I detached Beacon Broadcaster

**Comment by malcommac** [comment]
Merged in 5.1.0. Thank you :)
